api/scpca_portal/templates/readme_multiplexed.md

```diff
@@ -4,24 +4,34 @@ The [Single-cell Pediatric Cancer Atlas](https://scpca.alexslemonade.org) is a d
 
 ## Contents
 
-This download includes gene expression data from libraries processed using spatial transcriptomics and associated metadata for samples from project [{project_accession}]({project_url}) in the ScPCA portal.
+This download includes single-cell or single-nuclei gene expression files and associated metadata for samples from project [{project_accession}]({project_url}) in the ScPCA portal.
 
-For all spatial transcriptomics libraries (indicated by the `SCPCL` prefix) , a `SCPCL000000_spatial` folder will be nested inside the corresponding sample folder (`SCPCS` prefix) in the download.
-See the [FAQ section about samples and libraries](https://scpca.readthedocs.io/en/latest/faq.html#what-is-the-difference-between-samples-and-libraries) for more information.
+This project includes multiplexed samples, where multiple biological samples have been combined into libraries using cellhashing or similar technologies.
+The data files are divided into folders named by the set of samples that the libraries within contain (an underscore-separated list of sample ids, each with an `SCPCS` prefix).
```

Comment:
```suggestion
Each folder will be named with an underscore-separated list of sample ids (containing the `SCPCS` prefix) that correspond to the samples that are multiplexed and combined to create a given library(s). 
Within each folder are the gene expression data for each library containing the samples listed in the folder name.
```
I don't know if this is much better, but something about this sentence was confusing so I tried to clarify a bit?

Reply:
I really struggled with this sentence, and I didn't love where it ended up either. I tried again with the latest revision; let me know what you think!